inc/saifdb.h

```diff
@@ -165,6 +165,18 @@ typedef enum _sai_fdb_entry_attr_t
      */
     SAI_FDB_ENTRY_ATTR_ENDPOINT_IP,
 
+    /**
+     * @brief Attach a list of counters
+     *
+     * When it is empty, then packet hits won't be counted
+     *
+     * @type sai_object_list_t
+     * @flags CREATE_AND_SET
+     * @objects SAI_OBJECT_TYPE_COUNTER
+     * @default empty
+     */
+    SAI_FDB_ENTRY_ATTR_COUNTER_ID_LIST,
```

Comment:
do we want list here? I thought we discussed this last time, if we have list then we need to get know how many counters we can attach to an entry.

Reply:
Changed to a single ID